src/connectors/refinement-list/__tests__/connectRefinementList-test.js

```diff
@@ -98,10 +97,10 @@ describe('connectRefinementList', () => {
     });
 
     // test if widget is not rendered yet at this point
-    expect(rendering.callCount).toBe(0);
+    expect(rendering.mock.calls.length).toBe(0);
```

Comment:
You could use the shorthand method: `.not.toHaveBeenCalled()`

Reply:
I've switched to the number of call counts because it give a finer grain when there is a fail. In this case it would not matter much, but I still prefer consistency of API usage. Is that ok?